extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
Also considering the usage should be at the start of the repo's history one shouldn't lose the important tags?